library/std/src/io/buffered/mod.rs

```diff
@@ -14,6 +14,8 @@ use crate::io::Error;
 
 pub use bufreader::BufReader;
 pub use bufwriter::BufWriter;
+#[unstable(feature = "bufwriter_into_raw_parts", issue = "80690")]
```

Comment:
The feature gate name should change, as well.

Reply:
Well, perhaps, in theory, but that seems unnecessarily awkward to me.  In #80690 it looks like this is going ot be stabilised imminently, so changing the feature gate name will just lead to churn.

But if you say it should still be changed, despite that consideration, I'll do that.  Thanks.

Reply:
This seemed to be becoming a blocker so I have renamed it.